index.html

```diff
@@ -91,15 +91,27 @@ <h2>Languages</h2>
       <h2>Contact me</h2>
       <p>If you have an application you are interested in developing, a feature that you need built or
          a project that needs coding. I’d love to help with it </p>
-      <form action="https://formspree.io/f/meqnndkg" method="post">
-        <input type="text" name="name" placeholder="Yeremias" required maxlength="30">
-        <input type="email" name="email" placeholder="natajayanj14@gmail.com" required>
-        <textarea name="message" cols="30" rows="10" placeholder="Write your message here" required maxlength="500"></textarea>
-        <button class="btn" value="send">Get It Touch</button>
+      <form class="form" action="#" method="post">
+        <div class="input-control">
+          <input type="text" name="name" id="name" placeholder="Your name"  maxlength="30">
+          <div class="error"></div>
+        </div>
+        <div class="input-control">
+          <input type="email" name="email" id="email" placeholder="Your email address">
+          <div class="error"></div>
+        </div>
+        <div class="input-control">
+          <textarea name="message" id="message" cols="30" rows="10" placeholder="Write your message here" maxlength="500"></textarea>
+          <div class="error"></div>
+        </div>
+        <div class="button-wraper">
+          <button type="submit" class="btn" value="Submit">Get It Touch</button>
+        </div>
```

Comment:
Kindly make sure that the form has an action that is integrated with [this](https://formspree.io/html/) and it should have to work properly when the user submits email addresses all in lower case and should have to throw an error when the user tries to submit an email address with upper case. Please fix that.